src/SFA.DAS.PR.Jobs/MessageHandlers/EmployerAccounts/ChangedAccountNameEventHandler.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerAccounts.Messages.Events;
+using SFA.DAS.PR.Data;
+using SFA.DAS.PR.Data.Entities;
+using SFA.DAS.PR.Jobs.Models;
+
+namespace SFA.DAS.PR.Jobs.MessageHandlers.EmployerAccounts;
```

Comment:
Could use Tuples here.

Reply:
@DevenKShah I had initially use tuple but felt named properties would make the code more readable.

Reply:
Tuples can have named properties

Reply:
Example: 
```
(bool IsSuccess, string FailureMessage) IsValid() => (true, "hello world");

var (isSuccess, message) = IsValid(); 

var response = IsValid();

var isValid = response.IsSuccess;
```